draft-ietf-quic-datagram.md

```diff
@@ -85,7 +85,7 @@ when, and only when, they appear in all capitals, as shown here.
 
 Transmitting unreliable data over QUIC provides benefits over existing solutions:
 
-- Applications that open both a reliable TLS stream and an unreliable DTLS flow
+- Applications that want to use both a reliable stream and an unreliable flow
```

Comment:
I really like this new phrasing, though the comparison to TLS was useful, how about we tweak the last sentence of this paragraph to: `This can reduce the latency required for handshakes compared to opening both a TLS connection and a DTLS connection .`